docs/howto_config.md

```diff
@@ -157,3 +157,104 @@ Restart the MicroShift service to apply the manifests and verify that the `busyb
 sudo systemctl restart microshift
 oc get pods -n busybox
 ```
+
```

Comment:
It seems like this new content should be part of `docs/default_csi_plugin.md`. What do you think?

Reply:
Now it's in both files.

Reply:
I agree that we need to "split" the text:
* The design document should not contain the specifics of the configuration file internals. 
* The configuration file should list the options and explain them to the end-user

Reply:
For CNI, which also has a separate configuration file, we put the docs in docs/network/default_cni_plugin.md. Can we put all of the CSI docs in the CSI file, for consistency?

Reply:
Yes, we need to be consistent. 

I would prefer to have all of the generic/CSI/CNI configuration options documented at the same place in the howto_config.md file. I'm thinking about design as more of a stable text, while config options/details change more often.

Either way, if we choose to keep this text in the design, I think it's fine as long as we create links to the appropriate sections from the howto_config.md file. It would be nice to have all the configuration options reachable from a single doc.